src/pythermalcomfort/jos3_functions/thermoregulation.py

```diff
@@ -803,16 +808,16 @@ def ava_blood_flow(
         AVA blood flow rate at hand and foot [L/h].
     """
     # Cal. mean error body core temp.
-    cap_bcr = [10.2975, 9.3935, 13.834]  # Thermal capacity at chest, back and pelvis
+    cap_bcr = [10.2975, 9.3935, 4.488]  # Thermal capacity at chest, back and pelvis
```

Comment:
I trust you on these changes since I am not sure which one is the correct value. It would be nice to add a reference where you got the value from since aside from you only a few people in the world would be able to easily and quickly find the reference where you got these values from. The same is true from the change below in line 813.

Reply:
This seems to be a typo. It was pointed out that the values do not match those shown in the paper, so it has been corrected.